File: 009. Palindrome Number/isPalindrome.js
/**
 * @param {number} x
 * @return {boolean}
 */
 var isPalindrome = function(x) {
    if (x < 0) {
        return false
    }
    else {
        x = String(x)
        return x == x.split("").reverse().join("")
    }
};

// Runtime 310 ms / Memory 51.4 MB
// debug: https://jsfiddle.net/

File: 009. Palindrome Number/isPalindrome.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if (x < 0) { return false; }
        else {
            string str_x = to_string(x);
            string re_x = str_x;
            reverse(re_x.begin(),re_x.end());
            if (str_x == re_x) { return true; }
            else { return false; }
        }
    }
};